Rename: Konnect Get Started section (#5961)
Kong/docs.konghq.com: app/konnect/getting-started/deploy-service.md

title: Proxy and Test a Service
---

> Create a {{site.konnect_short_name}} Gateaway service to proxy your APIs. In this guide, you will create, proxy, and test a gateway service using Gateway Manager in {{site.konnect_short_name}}. 

Reviewer: ```suggestion
Create a {{site.konnect_short_name}} Gateaway service to proxy your APIs. In this guide, you will create, proxy, and test a gateway service using the {{site.konnect_short_name}} Gateway Manager. 
```

If you're following the {{site.konnect_short_name}} quickstart guide,
make sure you have

> [Configured a data plane node](/konnect/getting-started/configure-data-plane-node).

Reviewer: ```suggestion
[configured a data plane node](/konnect/getting-started/configure-data-plane-node).
```

## Verify the implementation

> If you used the Docker script to create a container

Reviewer: ```suggestion
If you used the Docker script to connect a data plane
```

earlier in [Configure a data plane node](/konnect/getting-started/configure-data-plane-node/),

> your control plane's default proxy URL is `localhost:8000`.

Reviewer: ```suggestion
your default proxy URL is `localhost:8000`.
```
